blocks/technology-table/technology-table.js

```diff
@@ -1,25 +1,26 @@
 function buildTechnologyTable(block) {
```

Comment:
@cuzuco2 the piece that is missing is that you can source an array from the node list and therefore use all the nice array functions. i'd re-write the function like this:

```
function buildTechnologyTable(block) {
  const rows = block.querySelectorAll('.technology-table > div');
  rows.forEach((cell, i) => {
    Array.from(cell.children)
      .forEach((child) => {
        if (i === 0) {
          const titleText = child.textContent;
          const title = document.createElement('h1');

          child.textContent = '';
          title.textContent = titleText;
          title.style.fontSize = 'var(--heading-font-size-l)';
          title.style.textAlign = 'left';

          child.appendChild(title);
          child.classList.add('techno-table-title');
        } else if (i === 1) {
          child.classList.add('techno-table-image');
        } else if (i === 2) {
          child.classList.add('techno-table-description');
        }
      });
  });
}
```

Reply:
Ah ok that was it, thanks